internal/ir/xds.go

```diff
@@ -779,6 +779,8 @@ type HTTPRoute struct {
 	CredentialInjection *CredentialInjection `json:"credentialInjection,omitempty" yaml:"credentialInjection,omitempty"`
 	// ExtensionRefs holds unstructured resources that were introduced by an extension and used on the HTTPRoute as extensionRef filters
 	ExtensionRefs []*UnstructuredRef `json:"extensionRefs,omitempty" yaml:"extensionRefs,omitempty"`
+	// BackendExtensionRefs holds unstructured resources that were introduced by an extension and used on the backendRef as a dynamic backend
+	BackendExtensionRefs []*UnstructuredRef `json:"backendExtensionRefs,omitempty" yaml:"backendExtensionRefs,omitempty"`
```

Comment:
curious why is this and UnstructuredRef are needed ?

Reply:
removed